src/index.ts

```diff
@@ -55,31 +55,19 @@ function getRecordExtractorSource(): string {
 }
 
 async function crawlerReindex(): Promise<void> {
-  const filteredCrawlers = [];
   let crawlerId = '';
-  let currentPage = 1;
-  let nbFetchedCrawlers = 0;
-  let crawlers: GetCrawlersResponseBody | undefined;
 
-  // Searching for the crawler, based on the name
-  do {
-    crawlers = await client.getCrawlers(100, currentPage++);
+  // Searching for the crawler, based on the name and application ID
+  const crawlers: GetCrawlersResponseBody | undefined =
+    await client.getCrawlers(100, 1, CRAWLER_NAME, ALGOLIA_APP_ID);
```

Comment:
To avoid passing unused parameters, you could modify the client method like that:
```
async getCrawlers({ itemsPerPage, page, name, appId } : {
  itemsPerPage?: number;
  page?: number;
  name?: string;
  appId?: string;
}): Promise<GetCrawlersResponseBody> {
```
And then call
```suggestion
    await client.getCrawlers({ name: CRAWLER_NAME, appId: ALGOLIA_APP_ID);
```
Wdyt?

Reply:
Like it :)